urlshortener/mainapp/templates/shortener/index.html

```diff
@@ -0,0 +1,11 @@
+{% extends "shortener/base.html" %} {% block content %} {% if url %}
+<section id="url_results">
+  <h3>Your chosen URL</h3>
+  <p>{{url.long_url}}</p>
+  <h3>Your shortened URL</h3>
+  <a>http://127.0.0.1:8000/{{url.short_url}}</a>
```

Comment:
You need to give this a href attribute for it to be clickable.

Reply:
oh yeah obviously